docs/microbit_micropython_api.rst

```diff
@@ -105,6 +139,34 @@ Each of these pins are instances of the ``MicroBitPin`` class, which offers the
     # returns boolean
     pin.is_touched()
 
+Except in the case of the pins marked **V2**, which offers the following API::
+    
+    **pin_logo**
+
+    # returns boolean
+    pin_logo.is_touched()
+
+    **pin_speaker**
+    
+    # disable the built-in speaker
```

Comment:
```suggestion
    # returns boolean
    pin_logo.is_touched()
    # disable the built-in speaker
```

I think it's fine to just include the code, or we should put `**pin_logo**` and ` **pin_speaker**` inside a comment.

Reply:
Done